schema/Makefile

```diff
@@ -16,3 +16,6 @@ validate: validate.go
 	go get -d ./...
 	go build ./validate.go
 
+clean:
```

Comment:
> If you do not add `.PHONY`, `make validate` can not be repeated…

Fair.  An example closer to your `clean` would be the [the `clean` entry in the main `Makefile`][1].

[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc5/Makefile#L97-L98